qdlt/tests/test_dltctrlmsg.cpp

```diff
@@ -22,7 +22,7 @@ TEST(CtrlPayload, parse) {
 
     auto ctx = app.ctxs[0];
     EXPECT_EQ(ctx.id, "MGR");
-    EXPECT_EQ(ctx.logLevel, '\xff');
-    EXPECT_EQ(ctx.traceStatus, '\xff');
+    EXPECT_EQ(ctx.logLevel, -1);
```

Comment:
I think, the problem is not here. I guess the byte layout generated by dlt-daemon on a big endian platform for the same message will be different, i.e. `data`-content for the same logLevel and traceStatus will be somewhat altered there.

Question: are you developing dlt-viewer on big-endian platform? If you are just using it there (and not developing), I do not think the tests matter.

Reply:
I am building dlt-viewer on Debian, so for both LE and BE architectures, and I would like to execute tests for all archs, to avoid missing regressions...
This is the arch lists:
https://buildd.debian.org/status/package.php?p=dlt-viewer&suite=experimental

Reply:
tbh, I do not completely understand why it fails on BE. `\ff` is decimal -1 for int8_t type. Do you understand the fix?

Reply:
not at all :)

Reply:
On other platforms, chars are 16 bit, so comparing 0xff to -1 will also fail.

Reply:
@RubenGarcia thanks for the info. Do you have an idea how the test can be rewritten so that it remains valid both on BE and LE platforms?